src/layout/Spacer/Spacer.tsx

```diff
@@ -8,18 +8,48 @@ export interface SpacerProps extends React.HTMLAttributes<HTMLDivElement> {
    */
   dense?: boolean;
 
+  /**
+   * If true, the spacer will be inline.
+   */
+  inline?: boolean;
+
   /**
    * The direction of the spacer.
    */
   direction?: 'row' | 'column';
+
+  /**
+   * How the content is arranged inside the spacer.
+   */
+  alignItems?: 'start' | 'end' | 'center';
+
+  /**
+   * How the content is arranged inside the spacer.
+   */
+  justifyContent?: 'start' | 'end' | 'center' | 'spaceBetween';
 }
 
 export const Spacer = React.forwardRef<HTMLDivElement, SpacerProps>(
-  ({ children, className, direction, dense, ...otherProps }, ref) => (
+  (
+    {
+      children,
+      className,
+      direction,
+      dense,
+      inline,
+      alignItems,
+      justifyContent,
+      ...otherProps
+    },
+    ref
+  ) => (
     <div
       className={classNames(css.container, className, {
         [css.dense]: dense,
-        [css[direction]]: direction
+        [css.inline]: inline,
+        [css[direction]]: direction,
+        [css[`align-${alignItems}`]]: alignItems,
```

Comment:
I would not use `-` in CSS Module class names.

Reply:
oh.. yea good call.